src/emulator/auth/index.ts

```diff
@@ -73,6 +73,12 @@ export class AuthEmulator implements EmulatorInstance {
         },
         configPath
       );
+    } else {
+      logger.logLabeled(
+        "WARN",
+        "auth",
+        `Error importing config from ${configPath}. Skipping config import`
+      );
```

Comment:
nit: wording: `Skipped importing config because ${configPath} does not exist.`

Reply:
Done.